facebook/react-native
# Support TLS connection for Android to RN DevServer

## Summary:
Following this proposal: https://github.com/react-native-community/discussions-and-proposals/issues/832

This PR will enable Android to be able to connect to a RN web server through TLS by setting "debug_http_tls"" as true

Currently, we're hardcoding the entire connection not having TLS. Which is different from iOS..

## Changelog:
[ANDROID] [ADDED] - Add TLS connection in Android layer for Dev server.

## Test Plan:
- Connect to a RN Web server that being hosted in TLS
- Check if it's able to connect.
